yourgov/components/Staff/StaffMembersTable.tsx

```diff
@@ -24,10 +28,44 @@ type StaffMembersTableProps = {
 
 const headingId = 'staff-members-heading';
 
+const successTypeToMessageText = {
+	invite: {
+		description: 'will receive an email asking them to create a staff account.',
+		titlePrefix: 'An email invitation has been sent to',
+		titleSuffix: '',
+	},
+	pause: {
```

Comment:
Do we need unpause as well?

Reply:
Probably, but not for the usability testing. I'd like to rewrite this logic at some point. It's almost outgrown its usefulness.